examples/e-commerce/package.json

```diff
@@ -12,7 +12,7 @@
     "classnames": "2.2.6",
     "instantsearch.js": "^4.5.0",
     "vue": "2.6.10",
-    "vue-instantsearch": "file:../../",
+    "vue-instantsearch": "3.5.0",
```

Comment:
changing this means that it doesn't run the local files anymore for e2e tests

Reply:
Right. But without this change examples does not works on codesandbox (and appears broken in the documentation).

We could make a separate e-commerce demo specifically for e2e tests, wdyt?

Reply:
we definitely could, or could use the netlify instead of codesandbox on the documentation, as used to work

Reply:
@Haroenv  I prefer having a codesandbox on the doc over the netlify because it enables people to modify it by themselves.
What if we modify the e2e script so that it adds `"vue-instantsearch": "file:../../"` on the fly?

Reply:
that seems like a reasonable solution, but how come we don't have this issue in InstantSearch.js for example? There we also use file:../../ right? 

Alternatively setting up a yarn workspace here would also solve it I think. Do you want to take over this PR @eunjae-lee ?

Reply:
Yeah, `yarn workspace` will definitely fix the issue. Haven't looked into why it doesn't happen in InstantSearch.js, though.
I'll take it over and will continue in the near future.

Reply:
looks like https://github.com/algolia/instantsearch.js/blob/master/examples/e-commerce/package.json#L24-L26 is used in InstantSearch.js, but it also doesn't work in codesandbox: https://codesandbox.io/s/github/algolia/instantsearch.js/tree/master/examples/e-commerce